x-pack/legacy/plugins/monitoring/README.md

```diff
@@ -72,7 +72,7 @@ cluster.
 1. Set the Kibana config:
   ```
   % cat config/kibana.dev.yml
-  xpack.monitoring.elasticsearch:
+  monitoring.ui.elasticsearch:
     url: "http://localhost:9210"
```

Comment:
This might be outdated. I don't think we support a single url anymore (or I can't find the right docs?)

Maybe better to use [hosts](https://www.elastic.co/guide/en/kibana/current/monitoring-settings-kb.html#monitoring-general-settings) instead?